src/components/activities/searchPatientActivity/types.ts

```diff
@@ -4,10 +4,11 @@ import { PatientDTO } from "../../../generated";
 export interface IStateProps {
   userCredentials: TUserCredentials;
   patientSearchResults: Array<PatientDTO> | undefined;
+  isLoading: boolean;
 }
 
 export interface IPatientSearchItemProps {
-  patient: any; //TODO: use generated type here
+  patient: PatientDTO;
```

Comment:
We have a task killer here, boys! It resolves the [OP-299](https://openhospital.atlassian.net/browse/OP-299).